### PR TITLE
week8

### PR DESCRIPTION
**Pull request by AigulY**
week8

### CODE REVIEW

File: 05-JWT-Basics/starter/errors/bad-request.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const CustomAPIError = require('./custom-error')
+const { StatusCodes } = require('http-status-codes');
+class BadReqest extends CustomAPIError {
```
Comment (aschey) [inline 05-JWT-Basics/starter/errors/bad-request.js]:
```suggestion
class BadRequest extends CustomAPIError {
```
Typo here, no big deal though

File: 05-JWT-Basics/starter/errors/bad-request.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  constructor(message) {
+    super(message)
+    this.statusCode = StatusCodes.BAD_REQUEST;
+  }
+}
+
+module.exports = BadReqest;
```
Comment (aschey) [inline 05-JWT-Basics/starter/errors/bad-request.js]:
```suggestion
module.exports = BadRequest;
```

File: 05-JWT-Basics/starter/errors/index.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const CustomAPIError = require('./custom-error')
+const BadRequestError = require('./bad-request')
+const UnauthenticatedError = require('./unauthenticated')
+
+module.export = {
```
Comment (aschey) [inline 05-JWT-Basics/starter/errors/index.js]:
```suggestion
module.exports = {
```
This needs to be `exports`

File: 05-JWT-Basics/starter/routes/main.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+const express = require('express')
+const router = express.Router();
+
+const { login, dashboard } = require('../controllers/main')
+
+const authMiddleware = require('../middleware/auth');
+const authentificationMiddleware = require('../middleware/auth');
```
Comment (aschey) [inline 05-JWT-Basics/starter/routes/main.js]:
```suggestion
```
This line is duplicated, you can remove this.

**Review by aschey — APPROVED** [review]

File: 05-JWT-Basics/starter/routes/main.js (hunk 4 above)
Comment (AigulY) [reply 05-JWT-Basics/starter/routes/main.js:7]: Hello! Thank you! Should I create new branch e.g. week8-1 to fix these mistakes or can I do changes to the same week8 assignment and git commit and push it again?

Comment (AigulY) [reply 05-JWT-Basics/starter/routes/main.js]: @aschey  Hello! Thank you! Should I create new branch e.g. week8-1 to fix these mistakes or can I do changes to the same week8 assignment and git commit and push it again?

Comment (aschey) [reply 05-JWT-Basics/starter/routes/main.js]: You can make the changes directly in this branch and then feel free to merge it afterwards.

Comment (AigulY) [reply 05-JWT-Basics/starter/routes/main.js]: @aschey oh, okay, got it! thank you!

**Pull request merged**